conda_smithy/cli.py

```diff
@@ -52,6 +53,25 @@ def generate_feedstock_content(target_directory, source_recipe_dir):
         with feedstock_io.write_file(forge_yml) as fh:
             fh.write("{}")
 
+    # merge in the existing configuration in the source recipe directory
+    forge_yml_recipe = os.path.join(source_recipe_dir, "conda-forge.yml")
+    yaml = YAML()
+    if os.path.exists(forge_yml_recipe):
+        feedstock_io.remove_file(
+            os.path.join(target_recipe_dir, "conda-forge.yml")
```

Comment:
This path is `forge_yml` above.

Reply:
forge_yml is `target_directory / "conda-forge.yml"`

Reply:
ack sorry!